service.go

```diff
@@ -64,16 +68,17 @@ func (s *locationServiceImpl) getLocationByUUID(uuid string) (location, bool) {
 	if !found {
 		return location{}, false
 	}
-	term, err := s.repository.getSingleLocationTaxonomy(rawId)
+	content, err := s.repository.GetTmeTermById(rawId)
 	if err != nil {
 		return location{}, false
 	}
-	return transformLocation(term), true
+	return transformLocation(content.(term), s.taxonomyName), true
 }
 
-func (s *locationServiceImpl) initLocationsMap(terms []term) {
-	for _, t := range terms {
-		tmeIdentifier := buildTmeIdentifier(t.RawID)
+func (s *locationServiceImpl) initLocationsMap(terms []interface{}) {
+	for _, iTerm := range terms {
+		t := iTerm.(term)
```

Comment:
Maybe should check that type assertion is correct:
`t, ok := iTerm.(term)`
`if !ok { ... }`